piecewise_web/js/ndt.js

```diff
@@ -8,20 +8,17 @@
 
 'use strict';
 
-var WebSocket;
-
-function NDTjs(server, serverPort, serverPath, callbacks, updateInterval, debugLevel) {
+function NDTjs(server, serverPort, serverProtocol, serverPath, callbacks,
+               updateInterval) {
   this.server = server;
   this.serverPort = serverPort || 3001;
   this.serverPath = serverPath || '/ndt_protocol';
+  this.serverProtocol = serverProtocol || 'ws';
```

Comment:
Was the intent of the conditionals to provide a fallback option for the non-secure websocket test to run if wss is not available?